packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -138,9 +145,10 @@ export function DevtoolsView(): React.ReactElement {
 	>();
 	const [queryTimedOut, setQueryTimedOut] = React.useState(false);
 	const [selectedTheme, setSelectedTheme] = React.useState(getFluentUIThemeToUse());
-
+	const [showMessage, setShowMessage] = React.useState(true);
```

Comment:
Nit: `showMessage` is a bit too generic. How about we name this `isBannerDismissed`? And flip the default value and the condition below, since now we would want to show things when it was false.